### ISSUE TITLE
org.json.JSONException: No value for username when trying to get username from json response

### ISSUE THREAD

**Issue by abishekvashok** [coding-blocks/Chatter#39]
```
org.json.JSONException: No value for username
        at org.json.JSONObject.get(JSONObject.java:389)
        at org.json.JSONObject.getString(JSONObject.java:550)
        at com.codingblocks.chatter.DashboardActivity$1.onResponse(DashboardActivity.java:80)
        at okhttp3.RealCall$AsyncCall.execute(RealCall.java:141)
        at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
        at java.lang.Thread.run(Thread.java:818)
```
is crashing the app

**Comment by abishekvashok** [comment]
Working on it

**Comment by abishekvashok** [comment]
Will make a pr when #38 is fixed. By the time see my fork's jack branch.

**Comment by chaostools** [comment]
#39 is already reolved. #42 is a different Issue. Please check my comment at that issue

**Comment by abishekvashok** [comment]
@championswimmer @the-dagger solved in #142 please review and add a bounty!

**Comment by abishekvashok** [comment]
@the-dagger @championswimmer I would appreciate a bounty for this :P
